packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -0,0 +1,600 @@
+        this._super(...arguments);
+        fooBarInstance = this;
+      },
+      foo: 'BORK',
+      barBinding: 'foo'
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{unbound foo}} {{unbound bar}}`
+    });
+
+    this.render(`{{foo-bar}}`);
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'OOF');
+      this.rerender();
+    });
+
+    this.assertText('OOF OOF');
+
+    this.runTask(() => set(fooBarInstance, 'foo', 'BLIP'));
+
+    this.assertText('OOF OOF');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'BORK');
+      this.rerender();
+    });
+
+    this.assertText('BORK BORK');
+  }
+
+  ['@test sexpr form does not update on parent re-render']() {
+    this.registerHelper('-capitalize', (args) => args[0].toUpperCase());
+
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super(...arguments);
+        fooBarInstance = this;
+      },
+      foo: 'bork'
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{-capitalize (unbound foo)}}`
+    });
+
+    this.render(`{{foo-bar}}`);
+
+    this.assertText('BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'oof');
+      this.rerender();
+    });
+
+    this.assertText('BORK');
+
+    this.runTask(() => set(fooBarInstance, 'foo', 'blip'));
+
+    this.assertText('BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'bork');
+      this.rerender();
+    });
+
+    this.assertText('BORK');
+  }
+
+  ['@test sexpr in helper format does not update on parent re-render']() {
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super(...arguments);
+        fooBarInstance = this;
+      },
+      foo: 'bork'
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{-capitalize (unbound (-doublize foo))}}`
+    });
+
+    this.registerHelper('-capitalize', (params) => params[0].toUpperCase());
+
+    this.registerHelper('-doublize', (params) => `${params[0]} ${params[0]}`);
+
+    this.render(`{{foo-bar}}`);
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'oof');
+      this.rerender();
+    });
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(fooBarInstance, 'foo', 'blip'));
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'bork');
+      this.rerender();
+    });
+
+    this.assertText('BORK BORK');
+  }
+
+  ['@test should be able to render an unbound helper invocation']() {
```

Comment:
(I know that they are direct ports, I just wasn't 100% sure what I am supposed to be looking out for here)